frontend/src/styles/Features.css

```diff
@@ -0,0 +1,26 @@
+:root {
+    --primaryTextColor: #eeeeee;
+    --secondaryColor: #777777;
+    --primaryColor: #555555;
+  }
```

Comment:
Was this brought over from the previous css file or did you add this?

Reply:
I added this. I thought it would make color scheming easier this way.

Reply:
We can change the colors. At the time, I didn't know what colors we wanted to work with.

Reply:
Ah ok cool since it deals with the primary css lets leave that in App.css. Wonderful job Rosie.

Reply:
^^ Can we move the root into App.css please.

Reply:
Yes!